content/hello-world-program.md

```diff
@@ -1,204 +1,233 @@
 # Hello World
 
-### List links for further reading here:
-
-- [https://docs.rs/solana-program/latest/solana_program/](https://docs.rs/solana-program/latest/solana_program/)
-
-# Objectives
+# Lesson Objectives
 
 *By the end of this lesson, you will be able to:*
 
+- Describe the Solana account model
 - Explain the entry point to a Solana program
-
 - Submit a transaction to invoke our “Hello, world!” program
 
 # TL;DR
 
 - **Accounts** on Solana each have a unique address, store data, and are owned by a program
-- **Programs** on Solana are a particular type of account marked as executable and store instruction logic
+- **Programs** on Solana are a particular type of account that is marked as executable and which stores instruction logic
 - **Transactions** on Solana are made up of instructions which are executed by programs
 
 # Overview
 
-Solana programs are written in the Rust programming language. Solana programs are stored in accounts with a unique address referred to as the program id. To understand programs on Solana we must first understand the basics of Solana’s Account Model. Once we understand Solana's account model, then we will be able to see how Solana programs (which are similar to what the Ethereum network calls, "smart contracts") are just a particular type of Solana account which can execute instructions.
+In this lesson we'll be using the Rust programming language to write a Solana program. We will also be using the Solana Playground to deploy a Solana program.
 
-### Account Model
+Solana programs are stored in accounts with a unique address. This unique address is referred to as the program id.
 
-As you'll recall from the Introduction to Reading Data lesson, accounts are like the files in Solana’s network ledger. All data stored on the Solana network are contained in what are referred to as accounts. Each account has its own unique address which is used to identify and access the account data. 
+To understand programs on Solana we must first understand the basics of Solana’s account model. Once we understand Solana's account model, then we will be able to see how Solana programs (which are similar to "smart contracts" on the Ethereum blockchain) are just a particular type of Solana account that can execute instructions.
+
+## Account Model
+
+As you'll recall from the [Read data from the network](./intro-to-reading-data.md) lesson, accounts are like the files in Solana’s network ledger. All data stored on the Solana network are contained in what are referred to as accounts. Each account has its own unique address which is used to identify and access the account data.
```

Comment:
Strike the reference to the prior lesson here. I'm guessing a lot of people will choose to go through *either* the client side lessons or the program lessons, so I don't think we should bank on them having read that lesson.